test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -61,6 +63,34 @@ var _ = Describe("Horizontal pod autoscaling (scale resource: CPU) [Serial] [Slo
 			scaleDown("rc", kindRC, rc, f)
 		})
 	})
+
+	Describe("ReplicationController light", func() {
+		It("Should scale from 1 pod to 2 pods", func() {
+			scaleTest := &HPAScaleTest{
+				initPods:   1,
+				cpuStart:   150,
```

Comment:
How about we change field names? I suggest 'totalInitialCPUUsage', 'perPodCPURequest' and 'targetCPUUtilizationPercent' (I'm open to different names). Each time I read these tests I'm confused with the names.